### DISCUSSION TITLE
help me , to do future ready internship....

### DISCUSSION THREAD

**Discussion by KONETIRAJASEKHAR562** [github/india#581] [GitHub Platform]
sir , I am recently joined the internship , please tell me how to do the project ih this? and in first module , Explore Azure compute services there is no other step for forward after this please guide me. Sir ,I'm a b.tech student ,doing first time to this internship word. please help me sir

**Comment by TheSuranaverse** [comment]
@KONETIRAJASEKHAR562 Please join the below server for any doubt or query you are having!

A group for all to get notified for any `Official GitHub Externship Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Link to join: https://discord.gg/ceMXzhfaka

**Comment by FRTMentor** [comment]
Please go through the some below links for more understanding on the Azure modules.
And also please go through the some YouTube links for more understanding.
https://www.youtube.com/watch?v=GuuNbzmve8Q
https://www.youtube.com/watch?v=A8U3FXUcR5k
https://www.youtube.com/watch?v=V4WtypyR9vE
Please develop a website with any platform which you're familiar.
And deploy it on the Azure.
